src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FileRouteSpec.scala

```diff
@@ -67,6 +70,7 @@ class FileRouteSpec extends TestContainerUtils with Matchers with TestRequest {
         1
       )
     }
+    val referenceMockServer = getReferencesMockSever(4)
```

Comment:
Should the number of references requested match the number of files, which is 5?

Reply:
The getReferencesMockServer method already creates a single reference. The parameter is how many additional references to create. I've renamed the method parameter to make it a bit clearer